### ISSUE TITLE
Is it possible to measure the total length of all lines in one dxf file?

### ISSUE THREAD

**Issue by madenney** [haplokuon/netDxf#173]
Let's say I was creating a program to determine how long it would take a CNC to cut out all lines in a file. First I would need the total length of all the lines. Is that possible with this software?

**Comment by haplokuon** [comment]
Yes you can, iterate through the list of lines of the drawing and add their lengths, if you need it you can use the Vector.Distance method.

**Comment by Kaio7912** [comment]
Please, write some code example. Thanks in advance!
